src/OpenTelemetry/.publicApi/net452/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
-static OpenTelemetry.Trace.TracerProviderExtensions.ForceFlush(this OpenTelemetry.Trace.TracerProvider provider, int timeoutMilliseconds = -1) -> bool
\ No newline at end of file
+static OpenTelemetry.Trace.TracerProviderBuilderExtensions.AddLegacyActivity(this OpenTelemetry.Trace.TracerProviderBuilder tracerProviderBuilder, string operationName) -> OpenTelemetry.Trace.TracerProviderBuilder
```

Comment:
Not a big fan of this name. How about `AddLegacySource`? Given we have `AddSource` already.

Reply:
I am okay with `AddLegacySource`

Reply:
we have opportunity to rename this until 1.1 stable is out, so i'd prefer to not block this pr.

Reply:
Fine by me, I'll stamp as approved.